File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.cs
public class Solution {
    public bool solution(string s) {
        bool answer = true;
        int len = s.Length;
        if(len != 4 && len != 6) return false;
        for(int i = 0; i < len; i++)
        {
            if(!char.IsDigit(s[i])) return false;
        }
        return answer;
    }
}

File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.cpp
#include <string>
#include <vector>

using namespace std;

bool solution(string s) {
    bool answer = true;
    for(char c : s)
    {
        if(!isdigit(c))
            answer = false;
    }
    if(s.size() != 6 && s.size() != 4)
        answer = false;
    
    return answer;
}